api/net/Areas/Admin/Controllers/ReportController.cs

```diff
@@ -173,7 +173,7 @@ public IActionResult Add(ReportModel model)
     [SwaggerOperation(Tags = new[] { "Report" })]
     public IActionResult Update(ReportModel model)
     {
-        var result = _reportService.UpdateAndSave(model.ToEntity(_serializerOptions));
+        var result = _reportService.UpdateAndSave(model.ToEntity(_serializerOptions, true));
```

Comment:
Update children objects in report.  I hope this doesn't introduce an issue when saving without content.